roles/openshift_node/tasks/main.yml

```diff
@@ -77,7 +77,7 @@
 - name: Check scheduleable state
   delegate_to: "{{ openshift_first_master }}"
   command: >
-    {{ openshift.common.client_binary }} get node {{ inventory_hostname }}
+    {{ openshift.common.client_binary }} get node {{ openshift_hostname }}
```

Comment:
@brenton talk to @detiber   He wrote those roles and might have a reason for not including it.